app/tasks/email_notifications.rb

```diff
@@ -66,14 +64,21 @@ def feedback_ending(time_text)
                        .pluck(:user_id))
 
     Ajat.info "feedback_ending|id:#{@contest.id}|time:#{time_text}"
-    send_emails(text: text, subject: subject, users: users)
+    send_emails(text: text, subject: subject, users: users, event: :feedback_ending)
   end
 
   private def send_emails(**hash)
-    hash[:users].pluck(:email).each_slice(200) do |arr|
-      Mailgun.send_message contest: @contest, text: hash[:text],
-                           subject: hash[:subject], bcc_array: arr,
-                           to: EMAIL_SINK
+    hash[:users].pluck(:email).each do |email|
+      user_id = User.find_by(email: email).id
+      notification_id = Notification.find_by(event: event).id
```

Comment:
Where is this `event` coming from?

Reply:
it's come from
```send_emails(text: text, subject: subject, users: users, event: :feedback_ending)```
i guess

Reply:
So isn't that supposed to be `hash[:event]`?

Reply:
yes, my bad